BBC-archive/psammead
# Locks us into Node v8.11.3

See https://github.com/BBC-News/simorgh/pull/1067 for more details

This PR aligns us all onto v8.11.3.

To test:

* checkout this branch
* `rm -rf node_modules package-lock.json`
* `npm install`
* `git diff` - should not be any different!
* `npm test` - should still pass

Note that the above testing instructions are time sensitive (i.e. if a dependency changes a subdependency version, we likely will have a diff).
